config/plugins/mdx-plugin/helpers.js

```diff
@@ -0,0 +1,110 @@
+import mdx from '@mdx-js/mdx';
```

Comment:
We don't currently publish this (see: https://github.com/FormidableLabs/spectacle/blob/task/rewrite/.npmignore) -- is this intended to be just for _our_ examples only or are we planning on releasing our MDX support?

/cc @carlos-kelly @SunburtReynolds